### DISCUSSION TITLE
在宝塔linux主机里如何安装第二个alist程序

### DISCUSSION THREAD

**Discussion by chouqiao** [AlistGo/alist#2121] [Q&A]
安装第二个后使用的依旧是5244端口，和第一个安装的冲突，如何指定端口号》

**Comment by imrelax** [comment]
修改第一个配置文件里面的端口号！然后再安装第二个。
或者停用第一个。安装第二个。然后停用。修改端口号再启用。
总之就是修改配置文件就OK了
config文件
  "port": 5244,

https://dalao.im/archives/482.html

**Reply by imrelax** [reply]
data 目录下的 config.json文件！

**Reply by chouqiao** [reply]
> data 目录下的 config.json文件！

感谢大佬回复，但是尝试下来会导致第一个站点502错误。
同一宝塔主机两个站点分别按照到两个二级域名，将1.xxx.com安装好后开ssl，改端口到5245，设置反向代理，正常访问，然后开始在2.xxx.com下开始安装，安装好后，第一个站点就502错误了。

**Reply by imrelax** [reply]
我自己就一个主机弄了好几个alist 肯定是没问题的
为了防止出错。 你可以直接把第一个网站就直接改端口。
比如第一个默认是5244  你改成5255之类的。 确保第一个是准确无误OK的。
然后第二个。 你改不改无所谓了。 防止出现问题。 你可以把端口号的跨度修改大一点！

**Reply by chouqiao** [reply]
我使用的是curl -fsSL "https://nn.ci/alist.sh" | bash -s install 加二级域名地址命令安装的

**Reply by imrelax** [reply]
宝塔的话 建议你手动安装吧！也不难。！就按照那个教程。主要是分三步。 
1、新建网站
2、下载文件赋权启动
3、设置守护进程 然后反向代理之类的
比较容易！

**Reply by chouqiao** [reply]
好的，感谢您答复，我继续试试

**Reply by chouqiao** [reply]
> 修改第一个配置文件里面的端口号！然后再安装第二个。 或者停用第一个。安装第二个。然后停用。修改端口号再启用。 总之就是修改配置文件就OK了 config文件 "port": 5244,
> 
> https://dalao.im/archives/482.html

大佬，这种情况下是不是只用设置守护第一个程序就行？

**Reply by imrelax** [reply]
我设置了2个！
